_posts/en/2019-01-20-rbf-in-the-wild.md

```diff
@@ -0,0 +1,243 @@
+
+### Xapo: Email Notifications for Unconfirmed Txs
+
+![Xapo emails showing incoming transactions screenshot](/img/posts/rbf-in-the-wild/rbf-xapo-email-notifications.png)
+
+Xapo sends email confirmations for both the original and replacement transaction which might indicate to a receiver that they will get 2,000 bits total via two transactions, instead of the actual 1,000 bits total in one transaction in this example.
+
+### Ledger Live: Error
+
+![Ledger Live's Portfolio showing red error message screenshot](/img/posts/rbf-in-the-wild/rbf-ledger-dashboard-sync-error-message.png)
+
+Ledger Live’s internal database was corrupted by the replacement transaction.
+
+Note: The fix was to reinstall Ledger Live. Optech is in communication with Ledger regarding this issue.
+
+### Electrum: Advanced Control of RBF Fee Bump
+
+![Electrums dialog of advanced fee bumping options including slider and textbox screenshot](/img/posts/rbf-in-the-wild/rbf-electrum-advanced-rbf-options.png)
+
+Electrum allows more advanced control of the fees in the UI. Additionally the dialog provides the capability to mark the transaction as final.
+
+### BRD: Generic Error Messages
+
+![BRD wallet transactions list screen showing a failed transaction screenshot](/img/posts/rbf-in-the-wild/rbf-brd-transaction-list-incoming-bumped.png)
+
+BRD shows a “Failed” label on the replacement transaction. In testing, the “Failed” transaction actually ends up being the transaction that confirms.
+
+## Usability Considerations
+
+A blanket recommendation for wallets, explorers, and other Bitcoin services would be to support Opt-in RBF, BIP125. Most of the services tested in this study do not have any support for RBF. As congestion builds on the network in the future, RBF will be an important tool in the scaling toolbox.
+
+Based on some of the examples above, there are some additional recommendations regarding RBF support.
+
+### Explorers Redirect to Replacement Transaction
+
+In almost all explorers, attempting to view the original transaction after the replacement confirms results in a 404 error. This can be confusing for users that have previously visited the original transaction's page on the explorer site and come back to find it has disappeared.
+
+Explorers should consider 301 redirects from the original to the replaced transaction that eventually confirms. Especially when the original transaction has been viewed by a visitor via that explorer. The redirection should be accompanied with an alert message indicating the original transaction has been replaced by current transaction.
+
+### Persist Original Transactions
+
+This recommendation is similar to previous one, only applied to wallets instead of explorers.
+
+Original transactions should be persisted in the UI, even subtly or under an advanced dialog. When sending a replacement transaction, a user may still want to have a reference to previously attempted, never-to-confirm transaction in their wallet.
```

Comment:
Do we want to add a note that this may not always be possible? If the wallet is a light client or not online when the original transaction was broadcast, it might only see the replacement.

Reply:
Yes! Thanks for bringing this up. Added wording around that.